algo/src/main/java/com/hartwig/serve/sources/ckb/CkbKnownEventsExtractor.java

```diff
@@ -196,4 +188,25 @@ private static Set<KnownFusion> convertToKnownFusions(@Nullable FusionPair fusio
                 CkbVariantAnnotator::annotateFusion,
                 variant);
     }
+
+    @NotNull
+    private static <T, U> Set<U> convertToKnownSet(@Nullable List<T> rawList, @NotNull Function<T, U> convert,
```

Comment:
Per advice, it would make this code more readable if you define that `U` has to extend `KnownEvent`. That way it makes things a bit more readable on what `e` could be in your filter function.

You do this by starting with `private static <T, U extends KnownEvent> convertToKnownSet(...`

Note that this has no actual code impact, just makes it clearer for reader what `U` could be.